content/geometry/object-environment/environment-structure.adoc

```diff
@@ -0,0 +1,482 @@
+
+.<Table title>
+[%header, cols="20, 80"]
+|===
+
+2+^| <Header>
+
+| *Origin*
+| ...
+
+| *x-axis*
+| ...
+
+| *y-axis*
+| ...
+
+| *z-axis*
+| ...
+|===
+
+
+Add a figure.
+
+.<Figure caption>
+
+== Vegetation
+
+Vegetation includes all organic growths or plants that are part of the landscape and grow on the terrain or on human-made structures, e.g., between roads and buildings. 
+
+.<Table title>
+[%header, cols="20, 80"]
+|===
+
+2+^| <Header>
+
+| *Origin*
+| ...
+
+| *x-axis*
+| ...
+
+| *y-axis*
+| ...
+
+| *z-axis*
+| ...
+|===
+
+
+Add a figure.
+
+.<Figure caption>
+
+== Other_Objects
+
+Apart from buildings, vegetation, and terrain, there are further objects part of the stationary surrounding. Some examples are bridges, highways, or tunnels. Road objects, e.g., signals, signs and traffic lights do not belong to this category, since they belong to the road network's Road Objects class, see below. 
+
+.<Table title>
+[%header, cols="20, 80"]
+|===
+
+2+^| <Header>
+
+| *Origin*
+| ...
+
+| *x-axis*
+| ...
+
+| *y-axis*
+| ...
+
+| *z-axis*
+| ...
+|===
+
+
+Add a figure.
+
+.<Figure caption>
+
+== Road_Network
+
+The Road Network includes the drivable area, sidewalks, and any other traffic objects such as traffic signals, signs, traffic lights or other traffic objects.
+
+.<Table title>
+[%header, cols="20, 80"]
+|===
+
+2+^| <Header>
+
+| *Origin*
+| ...
+
+| *x-axis*
+| ...
+
+| *y-axis*
+| ...
+
+| *z-axis*
+| ...
+|===
+
+
+Add a figure.
+
+.<Figure caption>
+//image::images/Vehicle_Structure_Door_Coord_Frame.svg[width=70%, scalewidth=10cm]
+
+== Drivable_Area
+
+The drivable area includes any surfaces that are part of the road structure including lanes and parking areas. It is comparable to the scope of the OpenDrive standard for on-road use cases.
+
+.<Table title>
+[%header, cols="20, 80"]
+|===
+
+2+^| <Header>
+
+| *Origin*
+| ...
+
+| *x-axis*
+| ...
+
+| *y-axis*
+| ...
+
+| *z-axis*
+| ...
+|===
+
+
+Add a figure.
+
+.<Figure caption>
+//image::images/Vehicle_Structure_Door_Coord_Frame.svg[width=70%, scalewidth=10cm]
+
+== Sidewalks 
+
+Sidewalks are traffic spaces that are dedicated for pedestrians and sometimes cyclists. 
+
+.<Table title>
+[%header, cols="20, 80"]
+|===
+
+2+^| <Header>
+
+| *Origin*
+| ...
+
+| *x-axis*
+| ...
+
+| *y-axis*
+| ...
+
+| *z-axis*
+| ...
+|===
+
+
+Add a figure.
+
+.<Figure caption>
+
+
+
+
+== Road_Marks
+
+
+Road marks include any markings on the road and traffic spaces.
+
+.<Table title>
+[%header, cols="20, 80"]
+|===
+
+2+^| <Header>
+
+| *Origin*
+| ...
+
+| *x-axis*
+| ...
+
+| *y-axis*
+| ...
+
+| *z-axis*
+| ...
+|===
+
+
+Add a figure.
+
+.<Figure caption>
+//image::images/Vehicle_Structure_Door_Coord_Frame.svg[width=70%, scalewidth=10cm]
+
+== Road_Object_<object_idx> (T)
+
+
+Road Objects include all other traffic items on or nearby the road, including, e.g., signals, signs, or traffic lights. 
```

Comment:
This is contradictive. Signals, signs and traffic lights should be excluded here because they would belong to the group Signals, Signs etc. (see below).